CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Setup
 
-To start developing on ICM Services, you'll need Golang v1.22.10.
+To start developing on with ICM Contracts, you'll need Solidity >= v0.8.25. [Foundry](https://book.getfoundry.sh/getting-started/installation) is the recommended development toolkit for working with Teleporter, and it comes bundled with the required Solidity version. To run the tests and linter locally, you'll need the dependencies described in [Setup](./README.md#setup), as well as [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
```

Comment:
"on with" is a typo here. But more broadly should this section just cover Contracts and not both?